src/applications/check-in/components/TravelPayOMB.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+// OMB Details
```

Comment:
We have a constants file somewhere where these usually go. I could see leaving them here if they aren't being used anywhere else though, but if they aren't being used anywhere else though they could just get hard coded at the attribute:  `<va-omb-info exp-date="11/30/2027" ...`

Reply:
These are specific to this form which is why I was reluctant to placing them else were as they felt like they belonged only here.

I did initially hard code them but since they are used in the children content it felt right to extract them to constants.